### PR TITLE
vim-patch: update colorschemes

### PR DESCRIPTION
**Pull request by clason**
- **vim-patch:fcc5346: runtime(colors): update included colorschemes**
- **vim-patch:d88ebcb: runtime(colors): update habamax scheme - tweak diff/search/todo colors**

(first patch somehow got missed?)

### CODE REVIEW

**Review by zeertzjq — CHANGES_REQUESTED** [review]

File: runtime/colors/retrobox.vim
[the remark this answers is not in the document]
Comment (clason) [reply runtime/colors/retrobox.vim:1]: Yeah, the script claims "1 chunk failed", but that's a big fat lie.

[the remark this answers is not in the document]
Comment (zeertzjq) [reply runtime/colors/retrobox.vim:1]: ?

**Review by zeertzjq — APPROVED** [review]

File: runtime/colors/retrobox.vim
[the remark this answers is not in the document]
Comment (clason) [reply runtime/colors/retrobox.vim:1]: Clearly, it failed to apply more than one individual hunk.

[the remark this answers is not in the document]
Comment (zeertzjq) [reply runtime/colors/retrobox.vim:1]: I think you read the message for the wrong file (e.g. quiet.vim where 1 hunk failed).

[the remark this answers is not in the document]
Comment (clason) [reply runtime/colors/retrobox.vim:1]: No, they all said "1 out of x hunks failed".

**Pull request merged**